app/src/main/java/com/arielwang/workoutlogger/features/track/ui/screen/TrackDestination.kt

```diff
@@ -0,0 +1,26 @@
+package com.arielwang.workoutlogger.features.track.ui.screen
+
+import androidx.compose.runtime.collectAsState
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavGraphBuilder
+import com.arielwang.workoutlogger.core.ui.screen.screenViewModel
+import com.arielwang.workoutlogger.navigate.NavigationDestination
+import com.arielwang.workoutlogger.navigate.destination.composable
+
+object TrackDestination: NavigationDestination {
+    private const val TRACK_ROUTE = "Track"
+
+    override fun route(): String = TRACK_ROUTE
+}
+
+internal fun NavGraphBuilder.addTrackScreen() {
+    with(TrackDestination) {
+        composable(this) {
+            screenViewModel<TrackViewModel>().let { viewModel ->
+ //               val state: TrackView.State by viewModel.uiState.collectAsState()
```

Comment:
Try not to commit this many commented-out codes into your PR. Just add them back when needed